### ISSUE TITLE
Order in diff vars are important!

### ISSUE THREAD

**Issue by latot** [sympy/sympy#11641]
Hi, well an unexpected thing:
```
>>> x=S("x")
>>> y=S("y")
>>> f=Function("f")(x, y)
>>> f.diff(x, y)
Derivative(f(x, y), x, y)
>>> f.diff(y, x)
Derivative(f(x, y), x, y)
```
This don't is true, to can be true have conditions to comply, but if you have only the ```f``` function you don't know if are true.
check this doc: https://en.wikipedia.org/wiki/Symmetry_of_second_derivatives

Thx. Cya.

**Comment by cbm755** [comment]
I agree, although I think its commonly done is CAS.  I just checked and Maple does this too for example.

In SymPy, it seems to be documented in `Derivative?` under "Ordering of variables"...

**Comment by cbm755** [comment]
My guess is this is "wontfix" (given that it is documented) but I'll let someone else confirm.

**Comment by latot** [comment]
mm, i see....
its necessary sort the variables in this cases?
i think would be great don't do that....

Thx. Cya.

**Comment by latot** [comment]
mm, thinking a little, sort this vars can break the evaluation of the function.... (seems a bad idea)
